src/ios/CDVInAppBrowser.h

```diff
@@ -76,6 +76,8 @@
 @property (nonatomic, copy) NSString* alphagradient2;
 @property (nonatomic, copy) NSString* toolbarcolor;
 @property (nonatomic, assign) BOOL hideallbuttons;
+@property (nonatomic) CGFloat toolbarheight;
+@property (nonatomic) CGFloat toolbarFooterHeight;
```

Comment:
Add you add some description in the README about the newly added toolbarheight and toolbarfooterheight option. Remember to clarify that it is only available on iOS. Also, change the camel case "toolbarFooterHeight" to "toolbarfooterheight" to keep it consistent with the other options